Attend-IN/app/(tabs)/home/index.js

```diff
@@ -26,9 +30,38 @@ function UserClasses () {
     }
   }, [userId])
 
+  useEffect(() => {
+    (async () => {
+      let { status } = await Location.requestForegroundPermissionsAsync()
+      if (status !== 'granted') {
+        setErrorMsg('Permission to access location was denied')
+        Alert.alert(
+          { errorMsg },
+          'This app requires location access. Please enable location services in your device settings.',
+          [{ text: 'OK', onPress: () => console.log('OK Pressed') }]
+        )
+        return
+      }
+
+      const locationSubscription = Location.watchPositionAsync(
+        { accuracy: Location.Accuracy.BestForNavigation, timeInterval: 30000, distanceInterval: 1 },
+        (newLocation) => {
+          setLocation(newLocation)
+        }
+      )
+      console.log(location.coords) // display user coordinates in termnial
+      return () => {
+        // Clean up the subscription when the component unmounts
+        locationSubscription.remove()
+      }
+    })()
+  }, [])
+
   return (
     <View style={styles.container}>
     <Text>User Classes</Text>
+    {/* <Text>{location?.coords.latitude}</Text>
+    <Text>{location?.coords.longitude}</Text> */}
```

Comment:
I think you forgot to remove this comment

Reply:
We Dont need it since we dont really need to show the user their current location. just wanted to leave it there so we could see how to get the lat and long